src/ai/backend/manager/container_registry/base.py

```diff
@@ -27,6 +32,20 @@
 
 log = BraceStyleAdapter(logging.getLogger(__name__))
 
+# generalize architecture symbols to match docker API's norm
+docker_arch_alias = {
+    'aarch64': 'arm64',
+    'arm64': 'arm64',
+    'x86_64': 'amd64',
+    'x64': 'amd64',
+    'amd64': 'amd64',
+    'x86': '386',
+    'x32': '386',
+    'i686': '386',
+    '386': '386',
+}
+manager_arch = docker_arch_alias.get(platform.machine(), platform.machine())
+
```

Comment:
Now lablup/backend.ai-common#119 has been merged and backported to 21.09.  Please update here to import it.